data/json/items/comestibles/egg.json

```diff
@@ -522,6 +522,76 @@
     "copy-from": "egg_fish",
     "rot_spawn": "GROUP_EGG_SALMON"
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "egg_cod",
+    "name": { "str_sp": "cod roe" },
+    "copy-from": "egg_fish",
+    "rot_spawn": "GROUP_EGG_COD"
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "egg_haddock",
+    "name": { "str_sp": "haddock roe" },
+    "copy-from": "egg_fish",
+    "rot_spawn": "GROUP_EGG_HADDOCK"
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "egg_bluefin_tuna",
+    "name": { "str_sp": "bluefin tuna roe" },
+    "copy-from": "egg_fish",
+    "rot_spawn": "GROUP_EGG_BLUEFIN_TUNA"
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "egg_halibut",
+    "name": { "str_sp": "halibut roe" },
+    "copy-from": "egg_fish",
+    "rot_spawn": "GROUP_EGG_HALIBUT"
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "egg_pollock",
+    "name": { "str_sp": "pollock roe" },
+    "copy-from": "egg_fish",
+    "rot_spawn": "GROUP_EGG_POLLOCK"
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "egg_flounder",
+    "name": { "str_sp": "flounder roe" },
+    "copy-from": "egg_fish",
+    "rot_spawn": "GROUP_EGG_FLOUNDER"
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "egg_mackerel",
+    "name": { "str_sp": "mackerel roe" },
+    "copy-from": "egg_fish",
+    "rot_spawn": "GROUP_EGG_MACKEREL"
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "egg_bluefish",
+    "name": { "str_sp": "bluefish roe" },
+    "copy-from": "egg_fish",
+    "rot_spawn": "GROUP_EGG_BLUEFISH"
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "egg_blackfish",
+    "name": { "str_sp": "black sea bass roe" },
+    "copy-from": "egg_fish",
+    "rot_spawn": "GROUP_EGG_BLACKFISH"
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "egg_porgy",
+    "name": { "str_sp": "scup (porgy) roe" },
```

Comment:
So I just realized you named the roe. Previously, all roe was unnamed because I figured most survivors wouldn't be able to tell them apart. I'd rather we kept it that way, if possible. Apologies for the late review, I didn't notice this yesterday.

Reply:
Fixed!